### PR TITLE
Add ImageVersion to the Json files

### PR DESCRIPTION
**Pull request by yl2558**
Add a provisioner to set the environment variable to VS2017, VS2019, WinCon and Ubuntu1604

### CODE REVIEW

File: images/win/WindowsContainer1803-Azure.json [hunk 1]
```diff
@@ -97,6 +98,12 @@
             "type": "windows-restart",
             "restart_timeout": "30m"
         },
+        {
+            "type": "powershell",
+            "inline": [
+              "setx ImageVersion {{user `image_version` }} /M"
```
[the remark this answers is not in the document]
Comment (yl2558) [reply images/win/WindowsContainer1803-Azure.json]: /m | Specifies to set the variable in the system environment. The default setting is the local environment.

[the remark this answers is not in the document]
Comment (alepauly) [reply images/win/WindowsContainer1803-Azure.json]: thanks, according to the [docs](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/setx), it should be ```/m``` but in the script you have ```/M```. It's often not the same in Windows commands.

File: images/linux/ubuntu1604.json [hunk 2]
```diff
@@ -18,7 +18,8 @@
         "installer_script_folder": "/imagegeneration/installers",
         "helper_script_folder": "/imagegeneration/helpers",
         "vm_size": "Standard_DS2_v2",
-        "capture_name_prefix": "packer"
+        "capture_name_prefix": "packer",
+        "image_version": null
```
[the remark this answers is not in the document]
Comment (yl2558) [reply images/linux/ubuntu1604.json]: The default value has been added.

File: images/win/WindowsContainer1803-Azure.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (yl2558) [reply images/win/WindowsContainer1803-Azure.json]: `/M` has been changed to `/m`.

**Comment by yl2558** [comment]
Tested VS2017, VS2019, WinCon, Ubuntu1604 on local machine with helper scripts `GenerateResourcesAndImage.ps1` and `CreateAzureVMFromPackerTemplate.ps1`. The ImageVersion environment variable has been added successfully to the VM.

**Pull request merged**